frontend/src/main-components/MyFoldersContent.js

```diff
@@ -39,14 +39,21 @@ class MyFoldersContent extends React.Component {
       justifyContent: "space-around",
       alignItems: "start",
     };
+
     let headingText =
       "You have no folders at the moment. Please make a new folder by clicking on the side menu.";
     if (!this.state.isDataFetched) {
       return <h1>loading</h1>;
     }
+
     if (this.state.folders) {
-      headingText = `You have ${this.state.folders.length} Folders`;
+      if (this.props.headingText === "main") {
+        headingText = `You have ${this.state.folders.length} Folders`;
+      } else {
+        headingText = this.props.headingText;
+      }
     }
```

Comment:
```js
let headingText;
if (this.state.folders) {
 ...
} else {
  headingText = "You have no folders..."
}
```

Reply:
I think this way of writing the assignment makes headingText more clear since it puts all the assignments in one place.

Reply:
done.